_posts/2018-07-27-my-apprenticeship.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "My Apprenticeship"
+date: 2018-07-27 12:0:00 +0200
+categories: culture how-we-learn
+author_name: Ugurcan Sengit
+author_url : /author/ugurcansengit
+author_avatar: ugurcansengit
+read_time : 12
+feature_image: posts/2018-04-20-holidaycheck-apprenticeship/apprenticeship.png
+---
+
+My days as an apprentice are slowly coming to an end. As the final day arrives, I have an irresistible desire to share. I would like to share how I felt, how I learnt, how I've been changed during my apprenticeship.
+
+It's really hard to structure everything flowing in my mind, so I'll follow the three phases I had during the apprenticeship.
+
+## The Basics
+
+Contrary to its name, this phase was the hardest part of the apprenticeship for me. During this phase, you need to learn important techniques about not just software development, but also human skills like communication and facilitation. Apart from the learnings, you need to set your mind to learn. It's easy to learn more about what you are currently doing, but by doing so you channel yourself more on the way you are already doing things. As you specialize more on a technique, you lost your flexibility and you create your own bubble. **The very first step I took was to wear the white belt**.
+
+### Wearing the White Belt
+
+I wanted to keep my flexibility while keep learning during my apprenticeship. But as I mentioned, it's not possible to see different opinions if you learn more about what you are already doing. After years of hands-on professional experience and much more social experience with the help of the [IEEE](https://www.ieee.org/), I almost always had an idea about the topic. To wear the white belt, you need to silence that inner voice. You need to talk less and observe more just to find out the different point of views. It's not as easy as sounds, during my life I saw so many people just can't observe. Knowing that every mind is unique helped me to observe. When you start to observe, you open yourself to the possibilities, new ideas and new techniques.
+
+When I started to observe, finally I wore the white belt. I figured out what needs to be learnt and I started to bake my knowledge with my new learnings.
```

Comment:
I would avoid word `bake` when you write about white belt. Just `expand` will be ok